LenovoLegionToolkit.Lib.Automation/LenovoLegionToolkit.Lib.Automation.csproj

```diff
@@ -13,6 +13,7 @@
 		<PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
 		<PackageReference Include="NeoSmart.AsyncLock" Version="3.2.1" />
 		<PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
+		<PackageReference Include="protobuf-net" Version="3.2.30" />
```

Comment:
Any particular reason why protobuf is needed? Wouldn't serialization to JSON not be enough?

Reply:
I tried but there were some problems I couldn't solve, so I chose the another way. If u perfer not to link a new dependency I'll give a second try.

Reply:
From what I see this is only 700KB so not a big deal. Was just curious.

Reply:
I see. But since all CLI related codes will be moved to another place I would also consider to do a minor refactor on how msgs are transfered. The current msg pack is only designed for Quick Action requests, and actually not so easy to extend. JSON is definitely a better choice but I know really few about the JSON lib LLT uses. I'll see what I can do tomorrow.